.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
       - 'docs'
 
 env:
-  BUILDER_VERSION: v0.9.32
+  BUILDER_VERSION: v0.9.31
```

Comment:
Trivial: builder version should be 0.9.32.

Reply:
That was intentional.  The 0.9.32 release did not complete container builds successfully.  Is there something that requires 0.9.32?

Reply:
It contains the following fix https://github.com/awslabs/aws-crt-builder/pull/214. @graebm recently updated the CI in https://github.com/awslabs/aws-crt-java/pull/577.